src/pyramid/settings.py

```diff
@@ -21,14 +21,17 @@ def aslist_cronly(value):
 
 
 def aslist(value, flatten=True):
-    """ Return a list of strings, separating the input based on newlines
+    """ Return a list. Separating the input based on newlines
     and, if flatten=True (the default), also split on spaces within
-    each line."""
+    each line when value is a string."""
```

Comment:
```suggestion
    Return a list, separating the input based on newlines.
    Also if ``flatten`` is ``True`` (the default), and if the line
    is a string, then the line will be split on spaces.
```

Line wrap as needed for docstrings.